pkg/skaffold/kubernetes/manifest/image_pull_policy.go

```diff
@@ -18,6 +18,10 @@ package manifest
 
 import apimachinery "k8s.io/apimachinery/pkg/runtime/schema"
 
+const imagePullPolicyField = "imagePullPolicy"
+const imagePullPolicyAlways = "Always"
```

Comment:
`import k8s "k8s.io/api/core/v1"`

and then you can use `k8s.PullIfNotPresent`` and `PullAlways`